chinese-text-search/tests/test_wikipediasearch.py

```diff
@@ -0,0 +1,82 @@
+__copyright__ = "Copyright (c) 2020 Jina AI Limited. All rights reserved."
```

Comment:
same, 2021

Reply:
I prefer to remove the tests folder. Since it's duplicate to current test